src/main/java/app/linguistai/bmvp/model/User.java

```diff
@@ -43,6 +45,14 @@ public class User {
     @Column(name = "language", nullable = false)
     private String currentLanguage;
 
+    @NotBlank
```

Comment:
This should be @NotNull but I don't think this field is necessary at all, why did you add this? Which problem does this field fix? I'm asking this because User model is used A LOT and need to be updated everywhere it is built, since this is marked as @NotBlank.

This has high potential to break things and I think we should not change model further, especially with NotNull and/or NotBlank fields. Can we think of a different solution to the problem that this fixes?